### DISCUSSION TITLE
Wix built installers on Windows11. Access denied on System32 folder

### DISCUSSION THREAD

**Discussion by ckearney-nitro** [wixtoolset/issues#7978] [Questions]
Hello

Is there way to elevate wix installers higer than admin level. I have notice that my installers are not working for windows 11. They work for windows 10. 

I am trying to copy some binary files into the system32 folder but I am getting access denied error. 
I did notice the default permissions have changed on the system32 folder between windows 10 to windows 11. 

Is there a way to handle this:
![Untitled](https://github.com/wixtoolset/issues/assets/90631217/7b8ef35c-07e0-4aa2-8bab-55b58511ed04)

**Comment by achainarong** [comment]
I have the exact same problem here. I build my own msi and had to install some prerequisites so i used the wix bundle to install both but i get access denied. Both MSI and the Setup.exe work independently.

**Comment by achainarong** [comment]
Somehow i don't have access into the 'C:\ProgramData\Package Cache' and the bundle can't cache the MSI there which leads to the problem that  the setup fails.

**Comment by chrpai** [comment]
Technically an administrator can take ownership of any object and then override the ACLs to get full control.  The question though is should you?  I'm see that folder has en_us localizations for microsoft components. I don't know enough to make a judgement call that it isn't absolutely unsafe/unsupported to put stuff in there but considering Microsoft took the effort to lock it down further I'd be asking myself if there isn't somewhere better to be installing my files.

**Reply by achainarong** [reply]
I just detected that bitdefender is blocking the path where the setup is trying to cache the msi and the prequisite exe for my installation. On a blank win 11 and win 10 vm its working flawelessly.
